src/main/res/values/strings.xml

```diff
@@ -110,20 +110,20 @@
     <string name="settings">Settings</string>
     <string name="processing_FORMAT">Processing %1$s…</string>
     <string name="project_website">Project website</string>
-    <string name="promotes_non_free_network_services">Promotes non-free network services</string>
-    <string name="promotes_non_free_software">Promotes non-free software</string>
+    <string name="promotes_non_free_network_services">Promotes non-libre network services</string>
+    <string name="promotes_non_free_software">Promotes non-libre software</string>
     <string name="provided_by_FORMAT">Provided by %s</string>
     <string name="proxy">Proxy</string>
-    <string name="proxy_host">Proxy host</string>
-    <string name="proxy_port">Proxy port</string>
-    <string name="proxy_type">Proxy type</string>
-    <string name="recently_updated">Recently updated</string>
+    <string name="proxy_host">Host</string>
+    <string name="proxy_port">Port</string>
+    <string name="proxy_type">Type</string>
+    <string name="recently_updated">Recently synced</string>
```

Comment:
Back to the latest-context, this is not right, it's about the apps that got recently updated.

Reply:
OK. Is it really "installed" though?

Reply:
Are you still on 0.4.3?
Nope, not all, the "updated" relates to the apps that got a new release.

Reply:
Ah. That is however a quirk in English.
Was hoping to make it unequivocal.